lib/elixir_analyzer/exercise_test/assert_call/compiler.ex

```diff
@@ -405,6 +408,7 @@ defmodule ElixirAnalyzer.ExerciseTest.AssertCall.Compiler do
         {:., _, [{:__MODULE__, _, _}, fn_name]} -> {module, fn_name}
         {:., _, [{:__aliases__, _, fn_module}, fn_name]} -> {fn_module, fn_name}
         {:|>, _, [_arg, {fn_name, _, atom}]} when is_atom(atom) -> {module, fn_name}
+        {:/, _, [{fn_name, _, atom}, _arity]} when is_atom(atom) -> {module, fn_name}
```

Comment:
The check in `matching_function_call?` has a guard to check if arity is an integer, but this one doesn't. Is that difference significant or accidental?

Reply:
I don't think the check is actually necessary, as I don't think `:/` will take any other type of argument. I think I left it in the first one for readability, but somehow didn't in this one. I'll add it here too.